R/get_ishares_index_data.R

```diff
@@ -52,7 +51,7 @@ get_ishares_index_data <- function(url, name, timestamp) {
     dplyr::mutate(
       base_url = .env$url,
       index_name = .env$name,
-      timestamp = .env$as_of_date
+      as_of_date = .env$as_of_date
```

Comment:
So we write the date used into the output dataset... I'm not sure if this will cause problems, I don't know if this column is ever actually used, or if it's just there for posterity

Reply:
It doesn't seem to directly be used anywhere in `workflow.prepare.pacta.indices`

Reply:
I'm pretty sure that's just baked in for transparency (which is good). We've been through these hoops before apparently: https://github.com/RMI-PACTA/pacta.data.scraping/pull/11/files

Reply:
Yeah I had a feeling this came from somewhere... it's a pity we don't have more context on what was causing a problem there before

Reply:
😞 my bad on not documenting that PR better